charms/filesystem-client/lib/charms/filesystem_client/v0/filesystem_info.py

```diff
@@ -499,6 +499,51 @@ def _get_or_create_auth_secret(self, model: Model) -> ops.Secret:
             )
         return secret
 
+@dataclass(frozen=True)
+class LustrefsInfo(FilesystemInfo):
+    """Information required to mount a LustreFS share."""
+
+    mgs_ids: [str]
+    """List of NID addresses for the MGS."""
+
+    fs_name: str
+    """Name of the exported filesystem."""
+
+    @classmethod
+    def from_uri(cls, uri: str, _model: Model) -> "LustrefsInfo":
+        """See :py:meth:`FilesystemInfo.from_uri` for documentation on this method."""
+        _logger.debug(f"FilesystemInfo.from_uri: parsing `{uri}`")
```

Comment:
Let's use the logging module's built-in formatter since it doesn't the string until absolutely necessary while f-strings will immediately format the message:
 
```suggestion
        _logger.debug(f"FilesystemInfo.from_uri: parsing `%s`", uri)
```